core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -225,18 +233,37 @@ var TopicEditorPage = function() {
     expect(await subtopics.count()).toEqual(count);
   };
 
-  this.addSubtopic = async function(title) {
-    await addSubtopicCard.click();
+  this.addSubtopic = async function(title, imgPath, htmlContent) {
+    await addSubtopicButton.click();
     await newSubtopicTitlefield.sendKeys(title);
+
+    await workflow.submitImage(
+      topicThumbnailButton, thumbnailContainer, imgPath, false);
+    var subtopicPageContentButton = element(by.css(
+      '.protractor-test-show-schema-editor'));
+    await waitFor.elementToBeClickable(subtopicPageContentButton,
+      'Edit subtopic htm content button taking too long to be clickable');
+    await subtopicPageContentButton.click();
+
+    var pageEditor = element(by.css(
+      '.protractor-test-create-subtopic-page-content'));
+    await waitFor.visibilityOf(pageEditor,
+      'Subtopic html editor takes too long to appear');
+    await (await browser.switchTo().activeElement()).sendKeys(htmlContent);
+
     await waitFor.elementToBeClickable(
       confirmSubtopicCreationButton,
       'Confirm subtopic creation button takes too long to be clickable');
     await confirmSubtopicCreationButton.click();
-    await waitFor.pageToFullyLoad();
+    await waitFor.invisibilityOf(
+      element(by.css('.new-subtopic-editor')),
```

Comment:
Can we use a `protractor-test-*` class here?

Reply:
Oops, sorry @U8NWXD I made the most basic mistake possible. So sorry. 
Fixed.